packages/rum-recorder/package.json

```diff
@@ -21,5 +21,9 @@
     "type": "git",
     "url": "https://github.com/DataDog/browser-sdk.git",
     "directory": "packages/rum-recorder"
+  },
+  "devDependencies": {
+    "@types/pako": "1.0.1",
+    "pako": "2.0.3"
```

Comment:
Why do we need pako for a the ViewEnd record?

Reply:
This is for tests: I want to make sure ViewEnd are inserted at the end of the record, so I need to inflate the request data to parse what would be sent to the intake.